Description: libfile-remove-perl FTBFS with Perl::MinimumVersion >=
 1.22. libfile-remove-perl uses functions in tests which requires Perl
 >= 5.6. Bump required version of Perl in META.yml.
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=52698
Bug-Debian: http://bugs.debian.org/560615
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
--- a/META.yml
+++ b/META.yml
@@ -21,7 +21,7 @@
   File::Glob: 0
   File::Path: 0
   File::Spec: 0.80
-  perl: 5.005
+  perl: 5.006
 resources:
   license: http://dev.perl.org/licenses/
 version: 1.42
